app/assets/stylesheets/_settings.scss

```diff
@@ -1 +0,0 @@
-$background-color--light: $color-grey-paler;
```

Comment:
Yeah, if you go to files changed view and then click the "34 changed files" link you can see it's been removed.

![](https://photos-6.dropbox.com/t/2/AADJuxbfFLchevAO2ndDTq4GBb2SNiwkITfT1SuHxOavYA/12/2535859/png/32x32/1/_/1/2/Screenshot%202015-09-09%2009.48.48.png/EOGK-gEY4sftQSAHKAc/9mZ73nH0G-YbrnDV4x55wvUda5XisA5VBpKorxnXCQo?size=1280x960&size_mode=2)